site/docs/v1/tech/apis/_tenant-request-body.adoc

```diff
@@ -261,6 +265,11 @@ Whether a parent email is required.
 [field]#tenant.familyConfiguration.parentRegistrationEmailTemplateId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.8.0#::
 The unique Id of the email template to use for parent registration.
 
+[field]#tenant.formConfiguration.adminUserFormId# [type]#[UUID]# [optional]#Optional# [since]#Available since 1.20.0#::
+The unique Id of the form to use for the Add and Edit User form when used in the FusionAuth admin UI. 
++
+A paid edition of FusionAuth is required to utilize custom forms. When this parameter is not provided it will default to a form provided by FusionAuth that may be used without a paid edition.
+
```

Comment:
Is this extra line required? You should be able to verify formatting when viewing the `https://site-local.fusionauth.io` - we should be able to have one empty line between this and the next `[field]`.

Reply:
This is the one empty line.

Reply:
I can remove it if you want though.

Reply:
Follow the same formatting of the existing field definitions, I don't think we have a trailing `+` on any of the other fields.

Reply:
So. To be clear... You want this...
```
[field]#tenant.formConfiguration.adminUserFormId# [type]#[UUID]# [optional]#Optional# [default]#defaults to **[see description]**# [since]#Available since 1.20.0#::
The unique Id of the form to use for the Add and Edit User form when used in the FusionAuth admin UI. 
+
A paid edition of FusionAuth is required to utilize custom forms. 
+
When this parameter is not provided, it will default to the form Id currently assigned to the Default tenant.

[field]#tenant.httpSessionMaxInactiveInterval# [type]#[Integer]# [optional]#Optional# [default]#default is `3600`# [since]#Available since 1.8.0#::
Time in seconds until an inactive session will be invalidated. Used when creating a new session in the FusionAuth OAuth frontend.
```

To be this?

```
[field]#tenant.formConfiguration.adminUserFormId# [type]#[UUID]# [optional]#Optional# [default]#defaults to **[see description]**# [since]#Available since 1.20.0#::
The unique Id of the form to use for the Add and Edit User form when used in the FusionAuth admin UI. 
+
A paid edition of FusionAuth is required to utilize custom forms. 
+
When this parameter is not provided, it will default to the form Id currently assigned to the Default tenant.
[field]#tenant.httpSessionMaxInactiveInterval# [type]#[Integer]# [optional]#Optional# [default]#default is `3600`# [since]#Available since 1.8.0#::
Time in seconds until an inactive session will be invalidated. Used when creating a new session in the FusionAuth OAuth frontend.
```

Reply:
I made this change. Should be resolved now.